sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 ### Breaking Changes
 
+- `PartitionClient::Close` now takes an optional `Azure::Core::Context` parameter to reflect that it now waits until the `Close` verb has fully completed.
+- `ProcessorPartitionClient::Close` now takes an optional `Azure::Core::Context` parameter to reflect that it now waits until the `Close` verb has fully completed.
```

Comment:
Purely from reading these lines, I don't see how adding an optional parameter is a breaking change.

Reply:
Adding the optional parameter isn't necessarily a breaking change. Changing the semantics of `Close` to block waiting on the response to the `DETACH` AMQP message may be.

I would rather be more conservative in breaking change notifications than assume that a behavior change won't have any side effects.